test/models/test_affiliation.py

```diff
@@ -87,3 +87,49 @@ def test_it_may_be_restricted():
 
     assert has.restricted is True
     assert has_not.restricted is False
+
+
+def test_it_can_get_combined_name_data():
+    address = Address(countries.get('gb'), 'City')
+    affiliation = Affiliation('1', address=address, organisation='Org',
+                              department='Dep', starts=datetime.now())
+
+    assert affiliation.get_name_list() == ['Dep', 'Org']
+
+
+def test_it_can_detect_if_current_without_ends_date():
+    start_date = datetime(2017, 1, 1, 1, 0, 0, tzinfo=timezone(timedelta(hours=1)))
+    address = Address(countries.get('gb'), 'City')
+    affiliation = Affiliation('1', address=address, organisation='Org',
+                              department='Dep', starts=start_date)
+
+    assert affiliation.is_current() is True
+
+
+def test_it_can_detect_if_current_with_ends_date():
+    start_date = datetime(2017, 1, 1, 1, 0, 0, tzinfo=timezone(timedelta(hours=1)))
+    ends_date = datetime(2100, 1, 1, 1, 0, 0, tzinfo=timezone(timedelta(hours=1)))
```

Comment:
I've been using `@freeze_time()` where-ever tests have meaningful dates. Means you don't have to set such things.

Reply:
I can't use `@freeze_time()` when also using `Affiliation.is_current()` as the `pendulum.instance.is_past()` uses `< datetime.datetime.now()` under the hood. Meaning it never thinks something is in the past.

If there are any other places I can use it I will add it.

Reply:
Ah yeah, forget how modules work! Could mock it to return the same value?

Reply:
I have used `@freeze_time()` where I am only looking for future dates (no past dates to check, so no problem) but to get it working in the use case mentioned I would need to to `@freeze_time()` then mock out the pendulum return value to be less than the `@freeze_time()` value as `is_past()` only accepts `< datetime.now()` as `True`. You trade off an extra variable being declared in the function with an extra decorator and a function arg for the mock. I think anywhere else is fine, but if we are actually testing `is_current()` its easier to just declare the variable.